src/home_robot_hw/home_robot_hw/remote/api.py

```diff
@@ -153,3 +157,34 @@ def dpt_cam(self):
     def get_frame_pose(self, frame, base_frame=None, lookup_time=None):
         """look up a particular frame in base coords"""
         return self._ros_client.get_frame_pose(frame, base_frame, lookup_time)
+
+    def go_to_manip_mode(self):
+        """Move the arm and head into manip mode."""
+        """
+        home_q = STRETCH_PREGRASP_Q
```

Comment:
What is all this code? Should it be deleted?

Reply:
used to switch the robot into manipulation posture; it was moved as part of refactor

Reply:
I meant the code in the comments. What is the code in the comment supposed to mean.
```
home_q = STRETCH_PREGRASP_Q
home_q = self.robot_model.update_look_at_ee(home_q)
self.goto(home_q, move_base=False, wait=True)
```

Reply:
i removed this

Reply:
If this code is not relevant anymore can we also delete it from the comments?

Reply:
oh wrong place. deleted now